tklinenums/tklinenums.py

```diff
@@ -271,18 +274,32 @@ def resize(self) -> None:
             end
         ) <= 1000 else self.config(width=temp_font.measure(f" {end} "))
 
-    def set_to_ttk_style(self) -> None:
-        """Sets the widget to the ttk style"""
+    def set_colors(self, background: str | None = None, foreground: str | None = None, ttk_style: bool | None = None) -> None:
```

Comment:
Does not need any arguments, since we have the color provider. But it is also called from the `<<ThemeChanged>>` binding, with a useless event object:

```suggestion
    def set_colors(self, event: object = None) -> None:
```